src/Stryker.Core/Stryker.Core/Initialisation/FindProjectComponentsCsharp.cs

```diff
@@ -0,0 +1,370 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.Extensions.Logging;
+using Stryker.Core.Exceptions;
+using Stryker.Core.InjectedHelpers;
+using Stryker.Core.MutantFilters.Extensions;
+using Stryker.Core.Options;
+using Stryker.Core.ProjectComponents;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+namespace Stryker.Core.Initialisation
+{
+    class FindProjectComponentsCsharp
+    {
+        private ProjectInfo _projectInfo;
+        private StrykerOptions _options;
+        private string[] _foldersToExclude;
+        private ILogger _logger;
+        private IFileSystem _fileSystem;
+
+        public FindProjectComponentsCsharp(ProjectInfo projectInfo, StrykerOptions options, string[] foldersToExclude, ILogger logger, IFileSystem fileSystem)
```

Comment:
```suggestion
        public CsharpProjectComponentsBuilder(ProjectInfo projectInfo, StrykerOptions options, string[] foldersToExclude, ILogger logger, IFileSystem fileSystem)
```

This reads better and explains more what the class does.